chevah/empirical
# [#1204] Allow preventing reactor.stop.

# Problem description

When using a threadpool for DB connection runDeferred will stop and close the pool.

This prevent running multiple deferred using the same pool.

In this case. Twistar Register still has a threadpool, but the pool is stop, so callbacks are never called.
# Changes description

An extra option 'prevent_stop' was added.
# How to try and test the changes

reviewers @alibotean 

Due to slow network I can not connect to the RO server.

For not this branch is only on github.

Here is an example usage: https://gist.github.com/adiroiban/5209471

The current 'prevent_stop' functionality is not 100% well implemented but it should help you to continue working on your branch. Hope the network will recover and I will polish this work.

Feel free to publish empirical 0.11.0
